libsplinter/Cargo.toml

```diff
@@ -95,6 +95,7 @@ experimental = [
     "biome-notifications",
     "biome-user",
     "circuit-template",
+    "registry-database",
```

Comment:
When stabilized, this feature should probably be removed and the feature guards replaced with `diesel` (at which point the combination of the `registry` and `postgres` and/or `sqlite` features will guard the implementation).